config/localization/en/tooltips.json

```diff
@@ -195,13 +195,13 @@
       "TITLE": "Reduce-Only",
       "BODY": "Reduce-Only prevents an order from changing the side of your position (long to short, or short to long). An order with Reduce-Only set will automatically resize and cancel itself as your position changes."
     },
-    "REDUCE_ONLY_TIMEINFORCE_IOC_FOK": {
```

Comment:
this is technically a breaking change but I guess that's fine since clients take these upgrades willingly and the compiler will let them know the keys don't exist anymore.